circle.yml

```diff
@@ -38,19 +38,20 @@ dependencies:
 test:
   override:
     - rvm $EARLY_STABLE --verbose do rake rubocop
-# Disabled because of a Java incompatibility
+# Disabled $JRUBY_VERSIONS tests because of a Java incompatibility
 # TODO: integration tests should run with the master branch of the agent
     - rvm $MRI_VERSIONS --verbose do rake test
-    - rvm $MRI_VERSIONS --verbose do appraisal contrib rake contrib
-    - rvm $MRI_VERSIONS --verbose do appraisal contrib rake monkey
-    - rvm $RAILS_VERSIONS --verbose do appraisal rails3-mysql2 rake rails
-    - rvm $RAILS_VERSIONS --verbose do appraisal rails3-postgres rake rails
-    - rvm $RAILS_VERSIONS --verbose do appraisal rails3-postgres-redis rake railsredis
-    - rvm $RAILS_VERSIONS --verbose do appraisal rails4-mysql2 rake rails
-    - rvm $RAILS_VERSIONS --verbose do appraisal rails4-postgres rake rails
-    - rvm $RAILS_VERSIONS --verbose do appraisal rails4-postgres-redis rake railsredis
-    - rvm $RAILS5_VERSIONS --verbose do appraisal rails5-mysql2 rake rails
+    - rvm $MRI_VERSIONS --verbose do appraisal contrib rake elasticsearch
```

Comment:
we should use the ``$RAILS_VERSIONS`` env variable like here: https://github.com/DataDog/dd-trace-rb/pull/66/files#diff-29944324a3cbf9f4bd0162dfe3975d88R46

We should wait Rails releases before supporting Ruby 2.4.0 and Rails.